src/filesystem/implementations/local.h

```diff
@@ -106,11 +112,18 @@ LocalFileSystem::IsDirectory(const std::string& path, bool* is_dir)
 {
   *is_dir = false;
 
+#ifdef _WIN32
+  std::wstring wpath = LocalizedPath::string2wstring(GetOSValidPath(path));
+  struct _stat64i32 st;
+  if (_wstat64i32(wpath.c_str(), &st) != 0) {
+    return Status(Status::Code::INTERNAL, "failed to stat file " + path);
+  }
+#else
   struct stat st;
-  if (stat(GetOSValidPath(path).c_str(), &st) != 0) {
```

Comment:
@ashishk98, why do we no longer need your `GetOSValidPath` helper function here for Windows long paths?